### ISSUE TITLE
Role creates /root/sshd directory on debian

### ISSUE THREAD

**Issue by Yannik** [willshersystems/ansible-sshd#264]
On debian (10/11/12), the `vars` file contains this line:

```
__sshd_runtime_directory: sshd
```

In `install.yml`, the following task is executed:

```
    - name: Make sure sshd runtime directory is present
      ansible.builtin.file:
        path: "{{ __sshd_runtime_directory }}"
        state: directory
        owner: root
        group: root
        mode: "{{ __sshd_runtime_directory_mode }}"
      when:
        - __sshd_runtime_directory is not none
```

This results in an directory `/root/sshd` being created. (Assuming ansible_user is root)

Obviously, this serves no purpose and should not be done.

**Comment by Jakuje** [comment]
Thanks for the report!

This was caused by some refactoring. When this task was introduced in 6b36488299e742138022c745a948d7c29e88ae1e the `__sshd_runtime_directory` was pointing to `/run/sshd`, which is normally created by systemd when starting the service and which is needed for the `validate` check to pass. But d6abe9c0325183180e6f83ba14c28781b9e6c6b9 changed the paths to relative (dropping the `/run/` prefix), which caused this bug. This should be fixed with #265

**Comment by Yannik** [comment]
@Jakuje Great, thanks. Looking forwarding to seeing #265 merged.

**Comment by Jakuje** [comment]
Merged as #265. We should get a next release soon. Closing.
